src/components/CodeAndPreview.tsx

```diff
@@ -35,7 +36,9 @@ export function CodeAndPreview({ snippet }: Props) {
       templateFromUrl.code
     )
   }, [])
-  const [manualEditsJson, setManualEditsJson] = useState("")
+  const [manualEditsJson, setManualEditsJson] = useState(
```

Comment:
If it's a string, we should make sure the variable name makes it clear it's a string.

```suggestion
  const [manualEditsFileContent, setManualEditsFileContent] = useState(
```